configs/bld-win64

```diff
@@ -0,0 +1,54 @@
+{
+    "hostname": "b-2008-ec2-%03d",
+    "us-east-1": {
+        "type": "bld-win64",
+        "domain": "build.releng.use1.mozilla.com",
+        "ami": "ami-fe051b96",
+        "subnet_ids": [ "subnet-2ba98340", "subnet-2da98346", "subnet-22a98349", "subnet-0822004e", "subnet-2da98346", "subnet-5bc7c62f", "subnet-7091d358" ],
+        "security_group_ids": [
+            "sg-18a07677",
+            "sg-f3927c9c"
+        ],
+        "instance_type": "m1.medium",
+        "distro": "win2008",
+        "ssh_key": "aws-releng",
+        "user_data_file": "/builds/aws_manager/cloud-tools/configs/bld-win64.cmd.user_data",
```

Comment:
The path shouldn't be absolute. It won't work on a local machine or if we decide to change the base directory.